.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish thundra-demo-todo-app-java to Docker Hub
+
+on:
+  pull_request:
```

Comment:
On pull request event is not the right for publishing. On push master could be and also on release event can be defined as a published type.